parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -814,6 +814,14 @@ configGPUDrivers() {
     if [ "$OS" = "$UBUNTU_OS_NAME" ]; then
         mkdir -p /opt/{actions,gpu}
         if [ "${CONTAINER_RUNTIME}" = "containerd" ]; then
+            # if target cloud is AzureUSGovernmentCloud, and NVIDIA_DRIVER_IMAGE contains GRID then use the specific 535 NVIDIA driver image
+            # this is being added because of the host driver in Fairfax is not compatible with the 550 guest drivers now
+            # It can be removed and synced with 550/other clouds, once it is moved to a compatible host driver (check with the HPC team)
+            if [ "$TARGET_CLOUD" = "AzureUSGovernmentCloud" ] && \
```

Comment:
Not necessarily now if this PR is in a rush. But I would recommend to add a shellspec unit test to cover all the if else conditions. Now it's getting complicated. With the tests, we can ensure (to some extent) that if later something are changed in this function, there will be no regression.

Reply:
right, not sure why I didn't request it ;)  if lack of testing is a concern, it's even more crucial to add the required unit tests no ?